src/sfx/AudioSource.cpp

```diff
@@ -1,10 +1,19 @@
 #pragma once
 
+#include "sfx.hpp"
 #include"AudioSource.hpp"
-#include "en/ComponentManager.hpp"
 
 namespace ari::sfx
 {
+	SoLoud::Wav wave;
+		
+
+	AudioSourceHandle LoadAudioSource(char* _path)
```

Comment:
Add all global functions to sfx.hpp and sfx.cpp

Reply:
So, Should I remove AudioSource.cpp/.hpp ?

Reply:
yes